tests/validation/tests/v3_api/resource/terraform/longliving/vsphere/v1/variables.tf

```diff
@@ -0,0 +1,23 @@
+variable vsphere_user {}
```

Comment:
I think we should include a type, default and description for each variable we declare even if it seems self-explanatory/obvious.